models/like.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require ('mongoose');
+const { Schema } = mongoose;
+
+const LikeSchema = new Schema ({
+    type: String,
+    author: { type: Schema.Types.ObjectId, ref: 'User' },
+    postItem: { type: Schema.Types.ObjectId, ref: 'Post' }
+});
+
+LikeSchema.index({"type": 1, "author": 1, "postItem": 1}, {unique: true});
```

Comment:
I'm not sure if we need an index on all of these fields.  what is the use case?  postItem makes sense to me but not sure about the others

Reply:
This is to make sure that each user can react in a given way on a given post only once.
I should have named the `author` column as `user` - it's the "author of the reaction", sorry! I have to make it more obvious!